### PR TITLE
Upgrade to gatsby v3

### PR DESCRIPTION
**Pull request by calcsam**
I've upgraded this starter to Gatsby v3 as part of my StrapiConf talk. The home page and article pages both get 12 additional points on Lighthouse due to the use of gatsby-plugin-image.

### CODE REVIEW

File: starter/src/pages/article/{StrapiArticle.slug}.js [hunk 1]
```diff
@@ -47,16 +43,27 @@ const Article = ({ data }) => {
...
-        <div
-          id="banner"
-          className="uk-height-medium uk-flex uk-flex-center uk-flex-middle uk-background-cover uk-light uk-padding uk-margin"
-          data-src={article.image.publicURL}
-          data-srcset={article.image.publicURL}
-          data-uk-img
-        >
-          <h1>{article.title}</h1>
+        <div style={{ display: "grid" }}>
+          <GatsbyImage
```
Comment (mfrachet) [inline starter/src/pages/article/{StrapiArticle.slug}.js:47]: Could you, please, add an "alt" attribute on this one two? 🙏🏻

**Review by mfrachet — COMMENTED** [review]
Hello Sam, it's good to see you there 😊 . Hope you're doing well!

Thanks for submitting this PR! I'm having one tiny detail about a missing "alt" attribute that throws on my dev tools. That would be amazing if you could fix this one 🙏🏻 

Apart from, that, it looks good to me 😊 .

Take care :D

File: starter/src/components/card.js [hunk 2]
```diff
@@ -24,9 +24,9 @@ const Card = ({ article }) => {
             <div className="uk-grid-small uk-flex-left" data-uk-grid="true">
               <div>
                 {article.node.author.picture && (
-                  <Img
-                    fixed={article.node.author.picture.childImageSharp.fixed}
-                    imgStyle={{ position: "static", borderRadius: "50%" }}
```
Comment (markkaylor) [inline starter/src/components/card.js:29]: could we keep the borderRadius here?

**Comment by markkaylor** [comment]
Hey @calcsam, I'm going to go ahead and merge this and then I will make the two small suggested changes.  Thanks again!

**Comment by calcsam** [comment]
Awesome, thanks @markkaylor !

**Pull request merged**